### PR TITLE
User story 23 facts screen

### PR DESCRIPTION
**Pull request by HermanForsberg**
Now the facts/wkik page is implemented in the main menu! It should show all the facts (and sources) with a subheader for the different trash we have implemented facts for. You should also be able to click the source to go to the webpage

### CODE REVIEW

**Review by HugoUnosson — APPROVED** [review]
Good job Herman, horrible choice of colors though, my eyes hurt

**Comment by HermanForsberg** [comment]
hahah fair enough

File: DAT257AgileProject/Assets/Scenes/Game/Main Menu.unity
Comment (Caramajau) [inline DAT257AgileProject/Assets/Scenes/Game/Main Menu.unity:1]: Unsure if this file should need to be changed, I believe you have made some updates only in the scene version of the prefab, is there any reason for this or did you just forget?

File: DAT257AgileProject/Assets/Scripts/Menus/WikiHandler.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        {
+            sourcesText = "Source: ";
+        }
+        else
+        {
+            sourcesText = "Sources: ";
+        }
+        foreach (SourceData source in sources)
+        {
+            sourcesText += $"\n- {source.SourceName} ({source.Date}) <i>{source.Title}</i> {source.Website}";
```
Comment (Caramajau) [inline DAT257AgileProject/Assets/Scripts/Menus/WikiHandler.cs:32]: You should also display the retrieval date in this menu, we is just adding retrieved ({{source.RetrievalDate}) to the string. (The reason I didn't do this in-game was to save space).

**Review by Caramajau — CHANGES_REQUESTED** [review]
Lastly, I think there is one important change that could be done, but if it is too difficult it could maybe be saved for future sprints. Which is that it would be nice for each subheading to have its own page with its own scrollbar instead of every fact being in a long scrollbar. Right now I find it quite difficult to navigate the menu. 

UPDATE: forgot to mention that I get this warning. Why?
warning_project

File: DAT257AgileProject/Assets/Scenes/Game/Main Menu.unity
Comment (HermanForsberg) [reply DAT257AgileProject/Assets/Scenes/Game/Main Menu.unity:1]: I forgot to update the prefab with some small last minute changes! The prefab is up to date now tho

File: DAT257AgileProject/Assets/Scripts/Menus/WikiHandler.cs (hunk 1 above)
Comment (HermanForsberg) [reply DAT257AgileProject/Assets/Scripts/Menus/WikiHandler.cs:32]: My thought was that this wasn't necessary, but I can add it if we want to show it. Not a hard change at all.

**Comment by HermanForsberg** [comment]
> Lastly, I think there is one important change that could be done, but if it is too difficult it could maybe be saved for future sprints. Which is that it would be nice for each subheading to have its own page with its own scrollbar instead of every fact being in a long scrollbar. Right now I find it quite difficult to navigate the menu.
> 
> UPDATE: forgot to mention that I get this warning. Why? warning_project

I didn't get the warning so I have no idea, I will look into it! And I agree that it would be a cleaner design, it would mean some restructuring. Maybe we can come back to that change if we feel that we have time over?

File: DAT257AgileProject/Assets/Scripts/Menus/WikiHandler.cs (hunk 1 above)
Comment (HermanForsberg) [reply DAT257AgileProject/Assets/Scripts/Menus/WikiHandler.cs:32]: now retrieval date is added

**Review by Caramajau — APPROVED** [review]

**Pull request merged**